index.html

```diff
@@ -1124,49 +1124,108 @@ <h1>
     <section>
       <h2>Contexts</h2>
 
-        <p>
+      <p>
 When two software systems need to exchange data, they need to use terminology
 and a protocol that both systems understand. The <code>@context</code>
 property ensures that two systems operating on the same DID document are using
-mutually agreed terminology.
-        </p>
+mutually agreed upon terminology.
+      </p>
 
-        <p>
+      <p>
+<a>DID Documents</a> contain a number of property-value pairs, where the
+properties are globally identified using <a>URIs</a>. However, those <a>URIs</a>
+can be long and and difficult to work with for humans; short-form aliases
+provide a more human-centric approach while not negatively impacting semantic
+accuracy. This specification uses the <code>@context</code> <a>property</a>
+to map such short-form aliases to the URIs used by specific <a>DID
+Documents</a>.
+     </p>
+
+      <p>
 <a>DID documents</a> MUST include the <code>@context</code> property.
-        </p>
+      </p>
 
-        <p class="note" title="The JSON-LD Context">
-More information about the
-<a href="https://www.w3.org/TR/json-ld/#the-context">JSON-LD Context</a>
-in general can be found in the [[!JSON-LD]] specification.
-        </p>
+      <dl>
+        <dt><dfn>@context</dfn></dt>
+        <dd>
+The value of the <code>@context</code> <a>property</a> MUST be an ordered set
+where the first item is a <a>URI</a> with the base context value
+<code>https://www.w3.org/ns/did/v1</code>. For reference, a copy of the base
+context is provided in Appendix <a href="#base-context"></a>. Subsequent items
+in the array MUST express context information and be composed of any combination
+of <a>URIs</a> or objects. It is RECOMMENDED that each <a>URI</a> in the
```

Comment:
```suggestion
of <a>URIs</a> and/or objects. It is RECOMMENDED that each <a>URI</a> in the
```
because `or` alone can be interpreted to mean that the array must be *all* URIs or *all* objects, rather than the intended meaning that a mix of these is acceptable.